marklogic-data-hub/src/main/java/com/marklogic/hub/step/RunStepResponse.java

```diff
@@ -195,4 +198,9 @@ public String toString() {
             stepDefinitionType, String.valueOf(success), status, totalEvents, successfulEvents, failedEvents,
             successfulBatches, failedBatches, stepStartTime, stepEndTime);
     }
+
```

Comment:
Can this method be moved to HubTestBase ? This doesn't have to be here

Reply:
Also a single test needs to have one run with "fullOutput: true" on in options to capture the fulloutput to make sure we're passing that back cleaning post mapping.

Reply:
@srinathgit sure. @aebadirad will we achieve the same results by testing with fullOutput set to true in other tests where we pass options?

Reply:
@aebadirad I know from the jnj slack channel that there is an issue with fullOutput: true. So dont think adding that in the test will add any value unless that bug is fixed.